apps/web/src/server/app/app.express.js

```diff
@@ -92,8 +91,12 @@ const njEnvironment = nunjucks.configure(viewPaths, {
 	express: app // the express app that nunjucks should install to
 });
 
-njEnvironment.addFilter('stripQueryParamsDev', stripQueryParametersDevelopment);
-njEnvironment.addFilter('className', className);
+for (const filterName in nunjucksFilters) {
+	if (Object.prototype.hasOwnProperty.call(nunjucksFilters, filterName)) {
```

Comment:
You are checking here to see if we added this already?

Reply:
Line 95? That's an eslint requirement to make sure we don't start adding .prototype values as filters